packages/api-client/src/api/getProducts/index.ts

```diff
@@ -14,8 +14,13 @@ export default async function getProducts({ client, config }: ApiContext, params
       include = 'default_variant,variants.option_values,option_types,taxons,images';
     }
 
+    const unique = Array.from(new Set(productPropertyFilters.map(filter => filter.productPropertyName)).values())
```

Comment:
Maybe you could call it more explicitly, something like `uniqueProductPropertyNames`? 
This function does quite a lot overall, so it would be great to avoid generic variable names